tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4412,4 +4412,382 @@ public function test_add_email_currency_conversion_notice_outputs_plain_text_for
+		$this->mock_gateway->process_payment( $order_id );
+
+		$this->assertNotNull( $captured_payment_info, 'Payment info should have been captured.' );
+
+		if ( $expect_full ) {
+			$this->assertArrayHasKey( 'statement_descriptor', $captured_payment_info );
+			$this->assertEquals( $expected_value, $captured_payment_info['statement_descriptor'] );
+		} else {
+			$this->assertArrayNotHasKey( 'statement_descriptor', $captured_payment_info );
+		}
+
+		if ( $expect_suffix ) {
```

PHPMD (2.15.0)

[error] The variable $expect_suffix is not named in camelCase. (undefined)

(CamelCaseVariableName)